app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,28 @@ def show
     @comments = @post.comments
     @likes = @post.likes
   end
+
+  def new
+    @current_user = current_user
+    @new_post = Post.new
+  end
+
+  def create
+    @new_post = Post.new(post_params)
+    @new_post.user = current_user
```

Comment:
Optional
- You can also go another way with setting the current user for a post by the following code instead of manually assigning the variable.
`@new_post = @current_user.posts.new(post_params)`

Reply:
Ok, I will try it 
Thanks for this tip 🙏